lib-injection/sitecustomize.py

```diff
@@ -28,9 +30,19 @@ def _log(msg, *args, level="info"):
     This function is provided instead of built-in Python logging since we can't rely on any logger
     being configured.
     """
-    if not debug_mode and level == "debug":
-        return
-    print("%s:datadog.autoinstrumentation(pid: %d): " % (level.upper(), os.getpid()) + msg % args, file=sys.stderr)
+    if level == "error":
+        priority = syslog.LOG_ERR
+    elif level == "warning":
+        priority = syslog.LOG_WARNING
+    elif level == "debug":
+        priority = syslog.LOG_DEBUG
+    else:
+        priority = syslog.LOG_INFO
```

Comment:
I guess defaulting to info doesn't but should we raise an Exception if level is set to an invalid value? This will allow us to catch regressions in tests.

Reply:
I was thinking we could solve this using literal types for the level to make it a type error if any other value is used. I do like your suggestion of raising on unexpected output though.